examples/dtls-sock/dtls-client.c

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       DTLS sock client example
+ *
+ * @author      Aiman Ismail <muhammadaimanbin.ismail@haw-hamburg.de>
+ */
+
+#include <stdio.h>
+
+#include "net/sock/udp.h"
+#include "net/sock/dtls.h"
+#include "net/ipv6/addr.h"
+#include "net/credman.h"
+
+#include "client_keys.h"
+
+/* TinyDTLS WARNING check */
+#ifdef WITH_RIOT_SOCKETS
+#error TinyDTLS is set to use sockets but the app is configured for socks.
```

Comment:
Please explain?

Reply:
This is copied from tinydtls example. Tinydtls has flag WITH_RIOT_SOCKET to use socket instead of the sock API but I just noticed this is still a TODO. So removed it for now.